hathor/transaction/storage/transaction_storage.py

```diff
@@ -94,6 +111,311 @@ def __init__(self):
         # Key storage attribute to save if the node has clean db
         self._clean_db_attribute: str = 'clean_db'
 
+        # Cache of block hash by height
+        self._block_height_index = BlockHeightIndex()
+
+        # Direct and reverse dependency mapping (i.e. needs and needed by)
+        self._dir_dep_index: Dict[bytes, _DirDepValue] = {}
```

Comment:
I feel we don't need this direct mapping. But we can discuss it further in the sync-v2 PR.

Reply:
Moved to #238